### PR TITLE
Make sure formatting is applied to std-bits projects

### PR DESCRIPTION
**Pull request by hubertp**
[ci no changelog needed]

### Pull Request Description

@radeusgd discovered that no formatting was being applied to std-bits projects.
This was caused by the fact that `enso` project didn't aggregate them. Compilation and
packaging still worked because one relied on the output of some tasks but
```
sbt> javafmtAll
```
didn't apply it to `std-bits`.

### Important Notes

Apart from `build.sbt` no manual changes were made.

### Checklist

Please include the following checklist in your PR:

- [ ] The documentation has been updated if necessary.
- [x] All code conforms to the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides.
- All code has been tested:
  - [ ] Unit tests have been written where possible.
  - [ ] If GUI codebase was changed: Enso GUI was tested when built using BOTH
        `./run.sh ide dist` and `./run.sh ide watch`.

### CODE REVIEW

**Review by JaroslavTulach — APPROVED** [review]

**Review by radeusgd — APPROVED** [review]

File: std-bits/table/src/main/java/org/enso/table/aggregations/Aggregator.java [hunk 1]
```diff
@@ -7,9 +7,7 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-/***
- * Interface used to define aggregate columns.
- */
+/** * Interface used to define aggregate columns. */
```
Comment (4e6) [inline std-bits/table/src/main/java/org/enso/table/aggregations/Aggregator.java]: Can you clean this file up a bit? A lot of comments were using triple-star syntax.

File: std-bits/table/src/main/java/org/enso/table/aggregations/CountDistinct.java [hunk 2]
```diff
@@ -10,16 +10,17 @@
 import java.util.List;
 import java.util.Set;
 
-/***
- * Aggregate Column counting the number of distinct items in a group.
- * If `ignoreAllNull` is true, does count when all items are null.
+/**
+ * * Aggregate Column counting the number of distinct items in a group. If `ignoreAllNull` is true,
```
Comment (4e6) [inline std-bits/table/src/main/java/org/enso/table/aggregations/CountDistinct.java]: And here

File: std-bits/table/src/main/java/org/enso/table/aggregations/CountEmpty.java [hunk 3]
```diff
@@ -6,17 +6,17 @@
 
 import java.util.List;
 
-/***
- * Aggregate Column counting the number of (non-)empty entries in a group.
- * If `isEmpty` is true, counts null or empty entries.
- * If `isEmpty` is false, counts non-empty entries.
+/**
+ * * Aggregate Column counting the number of (non-)empty entries in a group. If `isEmpty` is true,
```
Comment (4e6) [inline std-bits/table/src/main/java/org/enso/table/aggregations/CountEmpty.java]: This file too

File: std-bits/table/src/main/java/org/enso/table/aggregations/CountNothing.java [hunk 4]
```diff
@@ -5,17 +5,17 @@
 
 import java.util.List;
 
-/***
- * Aggregate Column counting the number of (not-)null entries in a group.
- * If `isNothing` is true, counts null entries.
- * If `isNothing` is false, counts non-null entries.
+/**
+ * * Aggregate Column counting the number of (not-)null entries in a group. If `isNothing` is true,
```
Comment (4e6) [inline std-bits/table/src/main/java/org/enso/table/aggregations/CountNothing.java]: And this

File: std-bits/table/src/main/java/org/enso/table/aggregations/First.java [hunk 5]
```diff
@@ -8,9 +8,7 @@
 import java.util.Comparator;
 import java.util.List;
 
-/***
- * Aggregate Column finding the first value in a group.
- */
+/** * Aggregate Column finding the first value in a group. */
```
Comment (4e6) [inline std-bits/table/src/main/java/org/enso/table/aggregations/First.java]: This

File: std-bits/table/src/main/java/org/enso/table/aggregations/GroupBy.java [hunk 6]
```diff
@@ -5,9 +5,7 @@
 
 import java.util.List;
 
-/***
- * Aggregate Column getting the grouping key.
- */
+/** * Aggregate Column getting the grouping key. */
```
Comment (4e6) [inline std-bits/table/src/main/java/org/enso/table/aggregations/GroupBy.java]: This

File: std-bits/table/src/main/java/org/enso/table/aggregations/Mean.java [hunk 7]
```diff
@@ -6,9 +6,7 @@
 
 import java.util.List;
 
-/***
- * Aggregate Column computing the mean value in a group.
- */
+/** * Aggregate Column computing the mean value in a group. */
```
Comment (4e6) [inline std-bits/table/src/main/java/org/enso/table/aggregations/Mean.java]: This should be the last one

File: std-bits/table/src/main/java/org/enso/table/aggregations/MinOrMax.java [hunk 8]
```diff
@@ -7,8 +7,9 @@
 import java.util.Comparator;
 import java.util.List;
 
-/***
- * Aggregate Column finding the minimum (minOrMax = -1) or maximum (minOrMax = 1) entry in a group.
+/**
+ * * Aggregate Column finding the minimum (minOrMax = -1) or maximum (minOrMax = 1) entry in a
```
Comment (4e6) [inline std-bits/table/src/main/java/org/enso/table/aggregations/MinOrMax.java]: No, probably this is the last one

File: std-bits/table/src/main/java/org/enso/table/aggregations/Mode.java [hunk 9]
```diff
@@ -8,9 +8,7 @@
 import java.util.List;
 import java.util.Map;
 
-/***
- * Aggregate Column computing the most common value in a group (ignoring Nothing).
- */
+/** * Aggregate Column computing the most common value in a group (ignoring Nothing). */
```
Comment (4e6) [inline std-bits/table/src/main/java/org/enso/table/aggregations/Mode.java]: There's a lot of them, apparently

File: std-bits/table/src/main/java/org/enso/table/aggregations/Percentile.java [hunk 10]
```diff
@@ -8,9 +8,7 @@
 import org.enso.table.data.table.Column;
 import org.enso.table.data.table.problems.InvalidAggregation;
 
-/***
- * Aggregate Column computing a percentile value in a group.
- */
+/** * Aggregate Column computing a percentile value in a group. */
```
Comment (4e6) [inline std-bits/table/src/main/java/org/enso/table/aggregations/Percentile.java]: This

File: std-bits/table/src/main/java/org/enso/table/aggregations/ShortestOrLongest.java [hunk 11]
```diff
@@ -7,9 +7,7 @@
 
 import java.util.List;
 
-/***
- * Aggregate Column finding the longest or shortest string in a group.
- */
+/** * Aggregate Column finding the longest or shortest string in a group. */
```
Comment (4e6) [inline std-bits/table/src/main/java/org/enso/table/aggregations/ShortestOrLongest.java]: And this

File: std-bits/table/src/main/java/org/enso/table/aggregations/StandardDeviation.java [hunk 12]
```diff
@@ -5,9 +5,7 @@
 import org.enso.table.data.table.Column;
 import org.enso.table.data.table.problems.InvalidAggregation;
 
-/***
- * Aggregate Column computing the standard deviation of a group.
- */
+/** * Aggregate Column computing the standard deviation of a group. */
```
Comment (4e6) [inline std-bits/table/src/main/java/org/enso/table/aggregations/StandardDeviation.java]: Last one, finally

**Review by 4e6 — APPROVED** [review]
All good, can you please clean up the comments that were starting with triple stars

**Review by jdunkerley — APPROVED** [review]

**Comment by hubertp** [comment]
@4e6 Thanks for noticing those

**Pull request merged**